src/kixi/comms/components/kafka.clj

```diff
@@ -194,23 +207,25 @@
   (attach-event-handler! [this group-id event version handler]
     (let [kill-chan (async/chan)
           _ (async/tap consumer-kill-mult kill-chan)]
-      (create-consumer (partial handle-result this :event)
-                       (process-msg? :event event version)
-                       kill-chan
-                       group-id
-                       topics
-                       broker-list
-                       handler)))
+      (->> (create-consumer (partial handle-result this :event)
```

Comment:
Obligatory frown on zealous use of thread-last in for non-sequential data. Don't really care.